### DISCUSSION TITLE
Resizing gameplay should be in pixels rather than percentages

### DISCUSSION THREAD

**Discussion by Bubbleman532** [ppy/osu#25053] [Ideas]
Currently the resize sliders only display a percentage. While this works, it's also imprecise, since multiple ticks on the slider are covered by one percentage value.

![image](https://github.com/ppy/osu/assets/117375368/cf8c95d9-0751-49db-9c86-ea926979b422)

A lot of players are used to having gameplay be within a set resolution value - I myself play on stable using 1600x900 letterboxing. Allowing that precision in this option would greatly ease transition between the two clients, and will also allow for finer control over effective gameplay area.

**Comment by peppy** [comment]
Also brought up at https://github.com/ppy/osu/discussions/24913.

The hard part for this is creating a good UX. Probably requires a custom control similar to the tablet one.
